projects/1.18/assets/xaeros-world-map/xaeroworldmap/lang/zh_cn.json

```
@@ -103,17 +103,17 @@
   "gui.xaero_right_click_box_map_settings": "§2%s§r打开设置",
   "gui.xaero_right_click_box_map_export": "将地图导出为PNG",
   "gui.xaero_waypoint_backgrounds": "渲染路径点背景",
-  "gui.xaero_right_click_map_cant_teleport": "§8（传送）模糊的Y坐标……",
+  "gui.xaero_right_click_map_cant_teleport": "§8（传送）不明确的Y坐标……",
   "gui.xaero_right_click_map_cant_teleport_world": "§8（传送）尚未确认……",
   "gui.xaero_right_click_map_cant_create_waypoint": "§8不能在这里设置路径点……",
-  "gui.xaero_no_world_map_item_message": "该模组被配置为需要你装备以下物品或放在快捷栏中：",
+  "gui.xaero_no_world_map_item_message": "该模组被配置为需要你装备以下物品或将其放在快捷栏中：",

```

这个长句子怎么看怎么怪……按感觉稍微改了一下
```suggestion
  "gui.xaero_no_world_map_item_message": "根据该模组配置，你需要将以下物品装备或放在快捷栏中：",
```